Rubberduck.Core/VersionCheck/IVersionCheck.cs

```diff
@@ -1,50 +1,12 @@
 ﻿using System;
-using System.Linq;
-using System.Net.Http;
-using System.Reflection;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace Rubberduck.VersionCheck
 {
     public interface IVersionCheck
     {
-        Task<Version> GetLatestVersionAsync(CancellationToken token = default(CancellationToken));
+        Task<Version> GetLatestVersionAsync(CancellationToken token = default);
```

Comment:
I wonder whether that's correctly defaulting to `CancellationToken.None` ...

Reply:
It does, the documentation says 
> You can also use the C# default(CancellationToken) statement to create an empty cancellation token.